src/kbmod_wf/task_impls/reproject_wu.py

```diff
@@ -12,3 +23,289 @@ def reproject_wu(input_wu=None, reprojected_wu=None, logger=None):
         f.write(f"Logged: {value} - {time.time()}\n")
 
     return reprojected_wu
+
+
+def reproject_wu(
+    original_wu_filepath: str = None,
+    uri_filepath: str = None,
+    reprojected_wu_filepath: str = None,
+    runtime_config: dict = {},
+    logger: Logger = None,
+):
+    wu_reprojector = WUReprojector(
+        original_wu_filepath=original_wu_filepath,
+        uri_filepath=uri_filepath,
+        reprojected_wu_filepath=reprojected_wu_filepath,
+        runtime_config=runtime_config,
+        logger=logger,
+    )
+
+    return wu_reprojector.reproject_workunit()
+
+
+class WUReprojector:
+    def __init__(
+        self,
+        original_wu_filepath: str = None,
+        uri_filepath: str = None,
+        reprojected_wu_filepath: str = None,
+        runtime_config: dict = {},
+        logger: Logger = None,
+    ):
+        self.original_wu_filepath = original_wu_filepath
+        self.uri_filepath = uri_filepath
+        self.reprojected_wu_filepath = reprojected_wu_filepath
+        self.runtime_config = runtime_config
+        self.logger = logger
+
+        self.overwrite = self.runtime_config.get("overwrite", False)
+        self.search_config = self.runtime_config.get("search_config", None)
+
+        # Default to 8 workers if not in the config. Value must be 0<num workers<65.
+        self.n_workers = np.max(1, np.min(self.runtime_config.get("n_workers", 8), 64))
+
+        # ! Decide if we want these from a runtime_config or exclusively read from the uri file
+        self.patch_corners = self.runtime_config.get("patch_corners", None)
+        self.guess_dist = self.runtime_config.get("guess_dist", None)
+        self.pixel_scale = self.runtime_config.get("pixel_scale", None)
```

Comment:
@wilsonbb the above three parameters were optional to pass in to the TNO script, but there was also logic that looked for the values in the uri.lst file. I would recommend that we consolidate on one of those two options, and not try to support both. 

I know that we have thorough control over the contents of the .lst files, but I don't know if we'll lose those with the work that Dino has done. 

Do you have thoughts?

Reply:
I think we want to have a discussion plan to finalize the file format, but the final result may be different from the current `.lst` file.

I'm personally feeling that we'll probably stick with just reading these from a file, but if you would prefer them to be solely passed as required arguments, I would probably be fine with that as well.

Reply:
I would advocate for reading these from the input file to avoid both human error and toil of updating a config file.